src/App.js

```diff
@@ -0,0 +1,66 @@
+import "react-native-gesture-handler";
```

Comment:
I knew changing this file will create issues. I suggest moving is back. It's giving errors

<img width="349" alt="Screenshot 2020-03-01 at 4 08 33 AM" src="https://user-images.githubusercontent.com/3650216/75616102-e2efff00-5b72-11ea-9961-dc83ad82c714.png">